lib/vmpooler.rb

```diff
@@ -52,6 +53,13 @@ def self.config(filepath='vmpooler.yaml')
       parsed_config[:graphite]['prefix'] ||= 'vmpooler'
     end
 
+    # statsd is an addition and my not be present in YAML configuration
+    if parsed_config[:statsd]
+      if parsed_config[:statsd]['server']
+        parsed_config[:statsd]['prefix'] ||= 'vmpooler'
+      end
+    end
```

Comment:
This is defined before the code in `lib/vmpooler/api.rb` (lines 19-23), right? Are the changes to `parsed_config[:statsd]['prefix']` reflected in `Vmpooler::API.settings.config[:statsd]['prefix']`? Is there a relationship at all there? Or are we just suffering from side effects of the intermingling of model and presentation logic?